### PR TITLE
{Release} Upgrade to Azure CLI 2.50.0

### PR DESCRIPTION
**Pull request by azclibot**
Updated Azure CLI version.

### CODE REVIEW

**Comment by yonzhan** [comment]
Release

File: src/azure-cli/HISTORY.rst [hunk 1]
```diff
@@ -3,6 +3,151 @@
 Release History
 ===============
 
+2.50.0
+++++++
+
+**AKS**
+
+* [BREAKING CHANGE] Both json payload and table format changed for `az aks get-versions` (#26702)
+* `aks create`: Add condition to disable `--enable-msi-auth-for-monitoring` for service principle (#26528)
```
Comment (evelyn-ys) [inline src/azure-cli/HISTORY.rst:12]:
```suggestion
* `az aks create`: Add condition to disable `--enable-msi-auth-for-monitoring` for service principle (#26528)
```

File: src/azure-cli/HISTORY.rst [hunk 2]
```diff
@@ -3,6 +3,151 @@
 Release History
 ===============
 
+2.50.0
+++++++
+
+**AKS**
+
+* [BREAKING CHANGE] Both json payload and table format changed for `az aks get-versions` (#26702)
```
Comment (zhoxing-ms) [inline src/azure-cli/HISTORY.rst:11]:
```suggestion
* `az aks get-versions`: Both json payload and table format changed (#26702)
```

File: src/azure-cli/HISTORY.rst [hunk 3]
```diff
@@ -3,6 +3,151 @@
...
+**APIM**
+
+* `apim apim graphqlapi resolver create`: Add new command to create gql api resolver (#26144)
+* `apim apim graphqlapi resolver show`: Add new command to show gql api resolver (#26144)
+* `apim apim graphqlapi resolver list`: Add new command to show gql api resolver list (#26144)
+* `apim apim graphqlapi resolver delete`: Add new command to delete gql api resolver (#26144)
+* `apim apim graphqlapi resolver policy create`: Add new command to create resolver policy (#26144)
+* `apim apim graphqlapi resolver policy show`: Add new command to show resolver policy (#26144)
+* `apim apim graphqlapi resolver policy list`: Add new command to list resolver policies (#26144)
+* `apim apim graphqlapi resolver policy delete`: Add new command to delete a policy (#26144)
```
Comment (zhoxing-ms) [inline src/azure-cli/HISTORY.rst:23]:
```suggestion
* `az apim graphqlapi resolver create`: Add new command to create gql api resolver (#26144)
* `az apim graphqlapi resolver show`: Add new command to show gql api resolver (#26144)
* `az apim graphqlapi resolver list`: Add new command to show gql api resolver list (#26144)
* `az apim graphqlapi resolver delete`: Add new command to delete gql api resolver (#26144)
* `az apim graphqlapi resolver policy create`: Add new command to create resolver policy (#26144)
* `az apim graphqlapi resolver policy show`: Add new command to show resolver policy (#26144)
* `az apim graphqlapi resolver policy list`: Add new command to list resolver policies (#26144)
* `az apim graphqlapi resolver policy delete`: Add new command to delete a policy (#26144)
```

Comment (evelyn-ys) [inline src/azure-cli/HISTORY.rst]:
```suggestion
* `az apim graphqlapi resolver create`: Add new command to create gql api resolver (#26144)
* `az apim graphqlapi resolver show`: Add new command to show gql api resolver (#26144)
* `az apim graphqlapi resolver list`: Add new command to show gql api resolver list (#26144)
* `az apim graphqlapi resolver delete`: Add new command to delete gql api resolver (#26144)
* `az apim graphqlapi resolver policy create`: Add new command to create resolver policy (#26144)
* `az apim graphqlapi resolver policy show`: Add new command to show resolver policy (#26144)
* `az apim graphqlapi resolver policy list`: Add new command to list resolver policies (#26144)
* `az apim graphqlapi resolver policy delete`: Add new command to delete a policy (#26144)
```

File: src/azure-cli/HISTORY.rst [hunk 4]
```diff
@@ -3,6 +3,151 @@
...
+
+**Storage**
+
+* `az storage blob upload(-batch)/set-tier/copy start(-batch)`: Cold Tier GA (#26585)
+* `az storage blob download-batch`: When matching pattern, list blobs with prefix to reduce the number of list calls (#26692)
+* Fix #26673: `az storage account or-policy create`: Now print out server error (#26706)
+
+**Synapse**
+
+* [BREAKING CHANGE] `az command a`: Make some customer-facing breaking change (#26589)
```
Comment (evelyn-ys) [inline src/azure-cli/HISTORY.rst:149]:
```suggestion
* `az synapse workspace create/update`: Support workspace encryption and user-assignment management identity (#26589)
```

File: src/azure-cli/HISTORY.rst [hunk 5]
```diff
@@ -3,6 +3,151 @@
...
+**App Service**
+
+* Fix #21168: `az webapp deploy`: Call OneDeploy through ARM proxy if `--src-url` is provided (#26620)
+* Fix #26647: `az webapp show`: Remove duplicate IPs from outbound addresses (#26738)
+* Fix #25497: `az webapp deploy`: Fix extension parsing if `src-path` has multiple '.'s (#26709)
+
+**ARM**
+
+* `az managedapp definition create/update`: Add new parameter `--deployment-mode` to support setting deployment policy (#26604)
+* Bump up `managed applications` API version to `2019-07-01` (#26604)
```
Comment (zhoxing-ms) [inline src/azure-cli/HISTORY.rst:43]:
```suggestion
```

File: src/azure-cli/HISTORY.rst [hunk 6]
```diff
@@ -3,6 +3,151 @@
...
+* `az eventhubs eventhub create/update`: Event Hubs Capture MSI feature added to eventhub entity (#26715)
+
+**IoT**
+
+* `az iot hub route`: Hide the deprecated command, please use `az iot hub message-route` instead of it. (#26535)
+* `az iot hub routing-endpoint`: Hide the deprecated command, please use `az iot hub message-endpoint` instead of it. (#26535)
+
+**Key Vault**
+
+* Fix `az keyvault certificate show`: Show policy.x509CertificateProperties.subjectAlternativeNames correctly (#26530)
```
Comment (evelyn-ys) [inline src/azure-cli/HISTORY.rst:84]:
```suggestion
* Fix #26527: `az keyvault certificate show`: Show policy.x509CertificateProperties.subjectAlternativeNames correctly (#26530)
```

File: src/azure-cli/HISTORY.rst [hunk 7]
```diff
@@ -3,6 +3,151 @@
...
+* Fix #26638: `az network traffic-manager endpoint`: Declare `--min-child-endpoints`, `--min-child-ipv4` and `--min-child-ipv6` as integer type (#26641)
+* `az network dns`: Support DNSSEC configuration and DS/TLSA record set (#26727)
+
+**Packaging**
+
+* Add Debian Bookworm support (#26690)
+
+**RDBMS**
+
+* [BREAKING CHANGE] `az postgres flexible-server migration update`: Remove unsupported update parameters `--db-names` and `--overwrite-dbs` (#26720)
```
Comment (evelyn-ys) [inline src/azure-cli/HISTORY.rst:117]:
```suggestion
* `az postgres flexible-server migration update`: Remove unsupported update parameters `--db-names` and `--overwrite-dbs` (#26720)
```

File: src/azure-cli/HISTORY.rst [hunk 8]
```diff
@@ -3,6 +3,151 @@
...
+* `az sql midb move/copy`: Add new commands for Managed Database Move/Copy feature (#26694)
+
+**SQL VM**
+
+* Fix #2442969: `az sql vm enable-azure-ad-auth/validate-azure-ad-auth`: Workaround Graph API bug by using client side filtering upon failure (#26689)
+* `az sql vm update`: Add configuration options for new SQL Assessment pre-requisites MMA->AMA migration (#26755)
+
+**Storage**
+
+* `az storage blob upload(-batch)/set-tier/copy start(-batch)`: Cold Tier GA (#26585)
```
Comment (calvinhzy) [inline src/azure-cli/HISTORY.rst:143]:
```suggestion
* `az storage blob upload(-batch)/set-tier/copy start(-batch)`: Cold Tier GA, add new tier type `--tier cold` (#26585)
```

File: src/azure-cli/HISTORY.rst [hunk 9]
```diff
@@ -3,6 +3,151 @@
...
+* `az monitor metrics alert create`: Add `()` into `--condition` grammar (#26616)
+
+**MySQL**
+
+* `az mysql flexible-server import create`: Add new command to facilitate migrations from mysql single to flexible servers (#26606)
+* `az mysql flexible-server restore/geo-restore/replica cerate`: Support `--tags` (#26648)
+
+**NetAppFiles**
+
+* Anf 25576 resource query info (#26519)
```
Comment (kairu-ms) [inline src/azure-cli/HISTORY.rst:97]:
```suggestion
* `az netappfiles volume replication resume`: Add warning on action to re-sync replication volumes that if destination volume has quota rules they will be overwritten by the source volumes quota rules. (#26519)
```

File: src/azure-cli/HISTORY.rst [hunk 10]
```diff
@@ -3,6 +3,151 @@
...
+**SQL VM**
+
+* Fix #2442969: `az sql vm enable-azure-ad-auth/validate-azure-ad-auth`: Workaround Graph API bug by using client side filtering upon failure (#26689)
+* `az sql vm update`: Add configuration options for new SQL Assessment pre-requisites MMA->AMA migration (#26755)
+
+**Storage**
+
+* `az storage blob upload(-batch)/set-tier/copy start(-batch)`: Cold Tier GA (#26585)
+* `az storage blob download-batch`: When matching pattern, list blobs with prefix to reduce the number of list calls (#26692)
+* Fix #26673: `az storage account or-policy create`: Now print out server error (#26706)
```
Comment (calvinhzy) [inline src/azure-cli/HISTORY.rst:145]:
```suggestion
* Fix #26673: `az storage account or-policy create`: Now throw server error that was previous silently ignored. (#26706)
```

File: src/azure-cli/HISTORY.rst [hunk 11]
```diff
@@ -3,6 +3,151 @@
...
+* `az mysql flexible-server import create`: Add new command to facilitate migrations from mysql single to flexible servers (#26606)
+* `az mysql flexible-server restore/geo-restore/replica cerate`: Support `--tags` (#26648)
+
+**NetAppFiles**
+
+* Anf 25576 resource query info (#26519)
+
+**Network**
+
+* [BREAKING CHANGE] `az network application-gateway waf-policy`: Rename output property `applicationGatewayIpConfigurations` to `applicationGatewayIPConfigurations` to keep consistent with the name in API (#26602)
```
Comment (necusjz) [inline src/azure-cli/HISTORY.rst:101]:
```suggestion
```

**Review by zhoxing-ms — DISMISSED** [review]

**Review by wangzelin007 — APPROVED** [review]

**Review by evelyn-ys — APPROVED** [review]

**Review by yanzhudd — DISMISSED** [review]

**Review by necusjz — APPROVED** [review]

**Review by jiasli — APPROVED** [review]

**Review by kairu-ms — APPROVED** [review]

**Review by necusjz — APPROVED** [review]

**Pull request merged**